utxo/utxo.go

```diff
@@ -2332,3 +2332,68 @@ func (uv *UtxoVM) queryContractBannedStatus(contractName string) (bool, error) {
 	ctx.Release()
 	return false, nil
 }
+
+// QueryUtxoRecord query utxo record details
+func (uv *UtxoVM) QueryUtxoRecord(accountName string, displayCount int64) (*pb.UtxoRecordDetail, error) {
+	defaultUtxoRecordDetail := &pb.UtxoRecordDetail{
+		Header: &pb.Header{},
+	}
+
+	addrPrefix := fmt.Sprintf("%s%s_", pb.UTXOTablePrefix, accountName)
+	it := uv.ldb.NewIteratorWithPrefix([]byte(addrPrefix))
```

Comment:
Is iterator thread-safe for concurrent operations?

Reply:
The doc of goleveldb says that the returned DB instance is safe for concurrent use, meaning that all DB's methods may be called concurrently from multiple goroutine.